build.assets/grpc/Dockerfile

```diff
@@ -12,26 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.10-stretch
+# Documentation on getting started with protobuf:
+# https://github.com/gogo/protobuf#getting-started
+
+# Requires 1.9+
+FROM golang:1.16-buster
```

Comment:
Does this include Go 1.16.2 security patch https://groups.google.com/g/golang-announce/c/ZWvSr9XM4wM/m/t8wy_q5_AgAJ?

Reply:
Looks like this tag is kept up to date with the latest minor release. https://hub.docker.com/_/golang